### PR TITLE
Test set for scripts from Defects folder

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
ATF Test Scripts to check [#2352](https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2352)

This PR is **ready** for review.

### Summary
Added test set for running of the scripts for issues from releases 4 and 5

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by dboltovskyi** [comment]
@GetmanetsIrina We need to align this set with the latest changes introduced within `RemoteATF` and `WebEngine` features.
On a current develop [94f92bb7eb](https://github.com/smartdevicelink/sdl_core/commit/94f92bb7eb) I've got the following results: (Taking into account that the latest `develop` is used in scripts)
```
-- PROPRIETARY:
  ABORTED:
  ./test_scripts/Defects/4_6/842_App_does_not_activate_when_policies_are_disabled.lua

  FAILED:
  ./test_scripts/Defects/4_5/1893_ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua
  ./test_scripts/Defects/5_0/2457_Needed_fixes_sending_of_UpdateDeviceList_notification_on_device_connect.lua

-- EXTERNAL_PROPRIETARY:
  ABORTED:
  ./test_scripts/Defects/4_6/842_App_does_not_activate_when_policies_are_disabled.lua

  FAILED:
  ./test_scripts/Defects/5_0/2457_Needed_fixes_sending_of_UpdateDeviceList_notification_on_device_connect.lua

-- HTTP:
  ABORTED:
  ./test_scripts/Defects/4_6/842_App_does_not_activate_when_policies_are_disabled.lua 
  ./test_scripts/Defects/5_1/2874_SDL_sends_system_request_with_correlationid_-1_and_INAVLID_DATA.lua
  
  FAILED:
  ./test_scripts/Defects/4_5/1881_OnDriverDistraction_After_changing_HMIlevel_from_NONE.lua 
  ./test_scripts/Defects/5_0/2457_Needed_fixes_sending_of_UpdateDeviceList_notification_on_device_connect.lua 
  ./test_scripts/Defects/4_5/1880_DefaultAndRPCTimeoutGENERIC_ERROR.lua 
  ./test_scripts/Defects/5_0/1035_SDL_doesnt_send_REJECTED_code_to_mobile_app_when_activating_app_from_HMI_with_activate.lua  
  ./test_scripts/Defects/5_0/1218_Mandatory_parameter_spaceAvailable_is_not_provided_in_response_for_PutFile_request.lua  
  ./test_scripts/Defects/4_5/1912_Video_service_start_via_2_protocol.lua  
  ./test_scripts/Defects/5_0/1031_SDL_transfer_OnKeyboardInput_notification_to_not_active_App_when_there_is_no_active_PerformInteraction_KEYBOARD.lua 
  ./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_6_SDL_respond_ACK_from_not_protected_to_protected.lua  
  ./test_scripts/Defects/4_5/1873_UnsubscribeVD_OnVD_Parameters_empty_in_policy_table.lua 
  ./test_scripts/Defects/5_0/964_SDL_sends_to_mobile_APPLICATION_NOT_REGISTRED_in_setAppIcon_responce_if_HMI_responds_with_INVALID_DATA.lua 
  ./test_scripts/Defects/4_5/1893_ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua 
  ./test_scripts/Defects/5_0/2446_SDL_doesn_t_resume_HMILevel_when_DEACTIVATE_HMI_with_isActive_is_true_then_false.lua  
  ./test_scripts/Defects/4_5/1395_Resumption_data_IGN_OFF.lua 
```
Please notice some scripts for `HTTP` fails at the 2nd try.

**Comment by dboltovskyi** [comment]
@GetmanetsIrina Please notice there is instability with:
```
./test_scripts/Defects/4_5/1893_ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua
```

**Comment by GetmanetsIrina** [comment]
@dboltovskyi, created the issue for 1893 and commented the script in test set.

File: test_sets/Defects/Defects_releases_4_x_and_5_x.txt [hunk 1]
```diff
@@ -0,0 +1,121 @@
+;-----------------------------------------------------------------------------------------------------
+;Functionality:
+;-----------------------------------------------------------------------------------------------------
+;-----------------------------------------------------------------------------------------------------
+;Policies
+;-----------------------------------------------------------------------------------------------------
+;./test_scripts/Defects/4_5/1206_REQUEST_PTU_Trigger_PTU_failed_previous_IGN_ON.lua https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2332
+./test_scripts/Defects/4_5/1376_PTU_all_flows.lua - already exists in smoke set
```
Comment (JackLivio) [inline test_sets/Defects/Defects_releases_4_x_and_5_x.txt]:
```
Starting
Running ./test_scripts/Defects/4_5/1376_PTU_all_flows.lua - already exists in smoke set

ATF 2.2: error: undefined option: '-'.

```

File: test_sets/Defects/Defects_releases_4_x_and_5_x.txt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+;-----------------------------------------------------------------------------------------------------
+;-----------------------------------------------------------------------------------------------------
+;Stream:
+;-----------------------------------------------------------------------------------------------------
+./test_scripts/Defects/4_5/1912_Video_service_start_via_2_protocol.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_1_SDL_does_not_respond_NACK_Navi_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_2_SDL_does_not_respond_NACK_Audio_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_3_SDL_respond_NACK_from_protected_to_not_protected.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_4_SDL_respond_NACK_from_protected_to_protected.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_5_SDL_respond_NACK_from_not_protected_to_not_protected.lua
```
Comment (JackLivio) [inline test_sets/Defects/Defects_releases_4_x_and_5_x.txt]:
```
--- Test --------------------------------------------------------------------------------------------
[00:47:41,210] Start_first_Service_NOT_protected_and_start_Stream                                    [FAIL] (2027 ms)
 HMI notification Navigation.OnVideoDataStreaming: Times: The most allowed occurences boundary exceed
 HMI notification Navigation.OnVideoDataStreaming: ValidIf: 
params.available: expected: true, actual value: false
params.available: expected: true, actual value: false
params.available: expected: true, actual value: false
[00:47:43,237] Start_video_service_via_2_protocol_with_expectation_of_
```

File: test_sets/Defects/Defects_releases_4_x_and_5_x.txt [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+./test_scripts/Defects/5_1/2874_SDL_sends_system_request_with_correlationid_-1_and_INAVLID_DATA.lua
+;-----------------------------------------------------------------------------------------------------
+;-----------------------------------------------------------------------------------------------------
+;Stream:
+;-----------------------------------------------------------------------------------------------------
+./test_scripts/Defects/4_5/1912_Video_service_start_via_2_protocol.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_1_SDL_does_not_respond_NACK_Navi_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_2_SDL_does_not_respond_NACK_Audio_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_3_SDL_respond_NACK_from_protected_to_not_protected.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_4_SDL_respond_NACK_from_protected_to_protected.lua
```
Comment (JackLivio) [inline test_sets/Defects/Defects_releases_4_x_and_5_x.txt]:
```
[00:47:19,851] Start_Stream                                                                          [FAIL] (2800 ms)
 HMI notification Navigation.OnVideoDataStreaming: Times: The most allowed occurences boundary exceed
 HMI notification Navigation.OnVideoDataStreaming: ValidIf: 
params.available: expected: true, actual value: false
[00:47:22,652] Start_second_Protected_Service              
```

File: test_sets/Defects/Defects_releases_4_x_and_5_x.txt [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+./test_scripts/Defects/5_1/2862_SDL_sends_UnsubscribeWayPoint_request_to_HMI.lua
+./test_scripts/Defects/5_1/2874_SDL_sends_system_request_with_correlationid_-1_and_INAVLID_DATA.lua
+;-----------------------------------------------------------------------------------------------------
+;-----------------------------------------------------------------------------------------------------
+;Stream:
+;-----------------------------------------------------------------------------------------------------
+./test_scripts/Defects/4_5/1912_Video_service_start_via_2_protocol.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_1_SDL_does_not_respond_NACK_Navi_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_2_SDL_does_not_respond_NACK_Audio_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_3_SDL_respond_NACK_from_protected_to_not_protected.lua
```
Comment (JackLivio) [inline test_sets/Defects/Defects_releases_4_x_and_5_x.txt]: [FAIL] (2622 ms)
 HMI notification Navigation.OnVideoDataStreaming: Times: The most allowed occurences boundary exceed
 HMI notification Navigation.OnVideoDataStreaming: ValidIf: 
params.available: expected: true, actual value: false
params.available: expected: true, actual value: false
[00:47:01,624] Start_video_service_via_2_protocol_with_expectation_of_StartServiceNACK

File: test_sets/Defects/Defects_releases_4_x_and_5_x.txt (hunk 1 above)
Comment (dboltovskyi) [reply test_sets/Defects/Defects_releases_4_x_and_5_x.txt]: @JackLivio Comment removed in 076783d

**Comment by dboltovskyi** [comment]
@GetmanetsIrina Please notice the following scripts requires updates due to merge of fix for `#3279` issue in `sdl_core`
```
./test_scripts/Defects/5_0/2446_SDL_doesn_t_resume_HMILevel_when_DEACTIVATE_HMI_with_isActive_is_true_then_false.lua
./test_scripts/Defects/5_0/2670_4_GetInteriorVD_after_ignition_off.lua
```

**Comment by GetmanetsIrina** [comment]
@dboltovskyi , the issue has been created and scripts are commented in de87326

**Comment by GetmanetsIrina** [comment]
@JackLivio, please provide the information about your environment (SDL, ATF commits).
I can not reproduce the problem on 
SDL: https://github.com/smartdevicelink/sdl_core/tree/release/6.1.0
ATF: https://github.com/smartdevicelink/sdl_atf/commit/da7fa4195c3bc8162094372fa895f3762811bd08

**Comment by JackLivio** [comment]
@GetmanetsIrina All issues resolved, thank you!

**Pull request merged**